bundles/core/org.eclipse.smarthome.core.thing/src/main/java/org/eclipse/smarthome/core/thing/binding/ThingHandlerCallback.java

```diff
@@ -88,4 +88,12 @@
      */
     void channelTriggered(Thing thing, ChannelUID channelUID, String event);
 
+    /**
+     * Give advice to the framework whether automatic state updates should be sent for the given channel or not.
+     *
+     * @param channelUID
```

Comment:
A description should be added if it is ready for a final review (also if the comment is "the channel UID" only).
Same below and also in other classes...